### ISSUE TITLE
[syncfusion_flutter_gauges] GaugeAnnotation creating duplicate widget

### ISSUE THREAD

**Issue by NANDANUNNI** [syncfusion/flutter-widgets#1493]
**annotation widget is duplicated on the top left**
also , unable to select/move the pointer if it is under this widget.

![image](https://github.com/syncfusion/flutter-widgets/assets/49134387/7f7c7843-289e-4d91-99c4-e644e9c864c5)

```
annotations: <GaugeAnnotation>[
            GaugeAnnotation(
                angle: 90,
                axisValue: 0,
                positionFactor: 0.1,
                horizontalAlignment: GaugeAlignment.center,
                verticalAlignment: GaugeAlignment.center,
                widget: Text('9999999',style: const TextStyle(
                  color: Colors.deepOrangeAccent,
                  fontSize: 50,
                  fontWeight: FontWeight.bold,
                ),),
                // Container(
                //   alignment: Alignment.center,
                //   width: 200,
                //   height: 200,
                //   child: buildTimer(widget.minutes.toInt()),
                // )
              ),
          ],
```

**Comment by LokeshPalani** [comment]
Hi,


We have checked the reported issue regarding the duplication of the annotation widget on the left and unable to move the pointer if it is under the widget. We tried to replicate the reported issue at our end. Unfortunately, we were not able to reproduce the reported issue as we are not exactly sure in what scenario the issue is replicating. Therefore, we kindly request you to try to replicate the reported issue in the attached test sample below and revert to us. This will help us assist you in a better way.


Regards,
Lokesh Palani.

[gauge_519940.zip](https://github.com/syncfusion/flutter-widgets/files/13293369/gauge_519940.zip)
